.github/workflows/ci-go.yaml

```diff
@@ -56,13 +56,13 @@ jobs:
       - name: Run GitHub atlas extension
         run: |
           gh atlas init-action -R ${{ secrets.TEST_REPO }} --token=${{ secrets.TEST_ATLAS_TOKEN }} --dir-name="sync-action-integration-test" --driver="sqlite" "sqlite_dir" \
-          --config-path="atlas.hcl" --config-env="local"
+          --config-path="atlas.hcl" --config-env="local" --flow=migrate
```

Comment:
I think is should be subcommand, and not a flag.
i.e: ` gh atlas init-action migrate` and `gh atlas init-action schema`

Reply:
Oh nice. Let me change this.